packages/components/pie-icon-button/README.md

```diff
@@ -8,37 +8,137 @@
   </a>
 </p>
 
+# Table of Contents
+
+1. [Introduction](#pie-icon-button)
```

Comment:
I really like this addition to the readme, we should consider it for our future yeoman generator